proto/buf/alpha/registry/v1alpha1/plugin.proto

```diff
@@ -257,6 +263,17 @@ message DeletePluginRequest {
 
 message DeletePluginResponse {}
 
+message SetPluginContributorRequest {
+  // The ID of the plugin for which the user will be set a role.
+  string plugin_id = 1;
+  // The ID of the user for which to be set a role in the plugin.
+  string user_id = 2;
+  // The new role of the user in the plugin,
+  // unspecified role means to remove the user's role in the plugin.
```

Comment:
```suggestion
  // The role to assign to the user.
  // Setting UNSPECIFIED means removing the user's role.
```

Reply:
applied these suggested changes to both repository, plugin and template endpoints